src/component/NewComments.js

```diff
@@ -0,0 +1,45 @@
+import { postComment } from "../utils/api";
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+
+
+export const NewComment = ({  setGetComments, comments }) => {
+
+  const { article_id } = useParams();
+  console.log(article_id)
+  const [newComment, setNewComment] = useState("");
+
+
+  const handleSubmit = (e) => {
+
+    e.preventDefault();
+
+    setGetComments((comments)=>{
+
+      return [ {
+        article_id: article_id,
+        author: "grumpy19",
+        body: newComment,
+        comment_id: -1,
+        created_at: new Date().toISOString(),
+        votes: 0,
+      }, ...comments]
```

Comment:
It would be good to add the new comment to the start of the array to indicate that the comment has been posted to the user and stop them posting multiple times